src/resultsdisassemblypage.cpp

```diff
@@ -69,7 +69,7 @@ ResultsDisassemblyPage::ResultsDisassemblyPage(FilterAndZoomStack *filterStack,
 
     ui->searchTextEdit->setPlaceholderText(QLatin1String("Search"));
     ui->asmView->setSelectionMode(QAbstractItemView::SelectionMode::ExtendedSelection);
-
+    ResultsUtil::setupDisassemblyContextMenu(ui->asmView);
```

Comment:
again, multiple things in one patch

please remove this for now, we should instead use KTextEditor